Basic/ClientDriven/Assets/Scripts/NetworkObjectSpawner.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using Unity.Netcode;
+using UnityEngine;
+using UnityEngine.Networking;
+using Random = System.Random;
+
+public class NetworkObjectSpawner : NetworkBehaviour
```

Comment:
1. This class doesn't seem to be in a namespace, can you add it? It should be the same namespace other classes are into
2. Please use `internal` unless this class needs to be accessible from other namespaces

Reply:
That is a good point I noticed that the other spawner script is also not in a namespace. @fernando-cortez Which namespace do you think I should use?

Reply:
2DSpaceShooter, Invaders, and Client Driven aren't namespaced, for the most part. Our Addressables is the exception, but even that one has a generic namespace.

Boss Room has Unity.BossRoom.<domain>. So not established namespace naming at the moment.

I would give this a generic `Unity.Multiplayer.Samples.ClientDriven` for now, until we sort this out.

I will find the ticket I created, assign it to myself, and bump its priority so it goes into next sprint!

Reply:
I will add it to the script